### PR TITLE
Update shapes challenge

### PR DESCRIPTION
**Pull request by v-aru**

### CODE REVIEW

File: javascript/js-structure/js-structure_shapes/components/Circle/Circle.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { getRandomColor } from "../../utils/randomColor.js";
+
+export function Circle() {
+    const circleElement = document.createElement("div");
+    circleElement.classList.add("circle");
+    console.log('Reached circle function');
```
Comment (mgczarnievicz) [inline javascript/js-structure/js-structure_shapes/components/Circle/Circle.js:6]: Please delete console.log at the end.

File: javascript/js-structure/js-structure_shapes/components/Pentagon/Pentagon.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { getRandomColor } from "../../utils/randomColor.js";
+
+export function Pentagon() {
+    const pentagonElement = document.createElement("div");
+    pentagonElement.classList.add("pentagon");
+    console.log('Reached pentagon function');
```
Comment (mgczarnievicz) [inline javascript/js-structure/js-structure_shapes/components/Pentagon/Pentagon.js:6]: Please delete console.log at the end.

File: javascript/js-structure/js-structure_shapes/components/Square/Square.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { getRandomColor } from "../../utils/randomColor.js";
+
+export function Square() {
+    const squareElement = document.createElement("div");
+    squareElement.classList.add("square");
+    console.log('Reached square function');
```
Comment (mgczarnievicz) [inline javascript/js-structure/js-structure_shapes/components/Square/Square.js:6]: Delete console.log at the end

**Review by mgczarnievicz — COMMENTED** [review]
Well done! 😄 .

Delete all unnecessarily `console.log` at the end, please. 
Console.log is a nice way to see that everything is working and being fine, but at the end, please delete them. Yes, I am being picky 😄

File: javascript/js-structure/js-structure_shapes/components/Circle/Circle.js (hunk 1 above)
Comment (v-aru) [reply javascript/js-structure/js-structure_shapes/components/Circle/Circle.js]: Agreed, I had added them in while I was debugging. Removed them now :)

File: javascript/js-structure/js-structure_shapes/components/Pentagon/Pentagon.js (hunk 2 above)
Comment (v-aru) [reply javascript/js-structure/js-structure_shapes/components/Pentagon/Pentagon.js]: Agreed, I had added them in while I was debugging. Removed them now :)

File: javascript/js-structure/js-structure_shapes/components/Square/Square.js (hunk 3 above)
Comment (v-aru) [reply javascript/js-structure/js-structure_shapes/components/Square/Square.js]: Agreed, I had added them in while I was debugging. Removed them now :)

**Review by mgczarnievicz — APPROVED** [review]

**Pull request merged**